workers/event_catcher/event_catcher.rb

```diff
@@ -2,13 +2,16 @@
 
 class EventCatcher
   def initialize(ems, endpoint, authentication, settings, messaging, logger, page_size = 20)
-    @ems            = ems
-    @endpoint       = endpoint
-    @authentication = authentication
-    @logger         = logger
-    @messaging      = messaging
-    @page_size      = page_size
-    @settings       = settings
+    @ems             = ems
+    @endpoint        = endpoint
+    @authentication  = authentication
+    @logger          = logger
+    @messaging       = messaging
+    @page_size       = page_size
+    @settings        = settings
+    @filtered_events = settings.dig("ems", "ems_vmware", "blacklisted_event_names") || []
+
+    logger.info("Filtered events: [#{filtered_events.join(", ")}]")
```

Comment:
```suggestion
    logger.info("#{log_prefix} Filtered events: [#{filtered_events.join(", ")}]")
```

Reply:
Oh nvm, I thought log_prefix was a method - on that note, maybe it should be a method.

Reply:
:+1: yeah good call easy enough to pull out into a method, I'll make it an instance method we can add in ems id and/or name if we want